ui.R

```diff
@@ -2,99 +2,124 @@ library(shiny)
 library(shinydashboard)
 library(shinyBS)
 library(shinyjs)
+library(DT)
+
+## App Meta Data----------------------------------------------------------------
+APP_TITLE  <<- "Title of the app"
+APP_DESCP  <<- paste(
+  "Description of the app",
```

Comment:
You don't need to say "Description of app"; that is what you're supposed to provide.